module/module.go

```diff
@@ -177,7 +177,6 @@ func (m *Module) Start(ctx context.Context) error {
 	m.activeBackgroundWorkers.Add(1)
 	utils.PanicCapturingGo(func() {
 		defer m.activeBackgroundWorkers.Done()
-		defer utils.UncheckedErrorFunc(func() error { return os.Remove(m.addr) })
```

Comment:
I think this should remain here. The module process creates the socket file, and so should remove it. But it's okay to have a fallback function attempt to cleanup after it in the parent too.

Reply:
Actually, as this sometimes logs errors when the file is already missing, it may be worth expanding this slightly to only attempt the removal if the "correct" stop sequence didn't remove it already.

Reply:
Expanded to only attempt removal if the .sock file still exists.